### DISCUSSION TITLE
"Multiple root elements detected" when using a livewire component multiple times

### DISCUSSION THREAD

**Discussion by mustafaosta** [livewire/livewire#4244] [1. Help]
I'm using livewire version 2.7.2

I have a livewire component being used multiple times inside a parent component and I'm providing the key property to the component
`@livewire('child-component', key('some unique value'))`

when the parent component refreshes/rerenders the following warning will show up
```
Livewire: Multiple root elements detected. This is not supported. See docs for more information https://laravel-livewire.com/docs/2.x/rendering-components#returning-blade
```
when the key property is removed the warning won't show but this will stop the child component from reacting on the changes of the parent component

**Comment by joshhanley** [comment]
@mustafaosta please have a look at these existing discussions #3887 and #4015 on this issue, the warning is just a warning though, so it shouldn't impact the functioning of the app.

There is also an open PR #4153 with a potential fix.

Hope this helps!
